### ISSUE TITLE
The package product 'Alamofire' requires minimum platform version 10.0 for the iOS platform, but this target supports 8.0 (in target 'Moya' from project 'Moya')

### ISSUE THREAD

**Issue by acecilia** [Moya/Moya#1903]
While trying out `Moya 14.0.0-alpha.2` with `Xcode 11.0 beta 6 (11M392r)` SPM fetches all the dependencies correctly, but when building I am getting the following error:
```
error: The package product 'Alamofire' requires minimum platform version 10.0 for the iOS platform, but this target supports 8.0 (in target 'Moya' from project 'Moya')
```

My dependency specification is as follows:
```
.package(url: "https://github.com/Moya/Moya.git", from: "14.0.0-alpha.2")
```

**Comment by sunshinejr** [comment]
Hey @acecilia, we didn't have minimum platform versions specified yet, so might be the issue. Just merged a PR with that, I have another thing to merge and then we can release another alpha/beta so it should be fixed by then. Gonna comment here once it happens. Sorry for trouble!

**Comment by sunshinejr** [comment]
Hey @acecilia, I just released Moya 14.0.0-beta.1, can you try that out? Also, make sure you have a proper platform specified in `Package.swift`, like:
```swift
// swift-tools-version:5.1

import PackageDescription

let package = Package(
    name: "Test",
    platforms: [
        .macOS(.v10_12),
        .iOS(.v10),
        .tvOS(.v10),
        .watchOS(.v3)
    ],
    products: [
        .library(
            name: "Test",
            targets: ["Test"]),
    ],
    dependencies: [
        .package(url: "https://github.com/Moya/Moya.git", .exact("14.0.0-beta.1")),
    ],
    targets: [
        .target(
            name: "Test",
            dependencies: ["Moya"]),
        .testTarget(
            name: "TestTests",
            dependencies: ["Test"]),
    ]
)
```

**Comment by acecilia** [comment]
@sunshinejr Thanks for the message. I expected I would have some time to look at this, but I do not think it will happen anytime soon, thus closing it.
